tests/unit/lib/test_opensearch_config.py

```diff
@@ -172,11 +171,5 @@ def test_set_node_and_cleanup_if_bootstrapped(self, gethostbyaddr):
         opensearch_conf = self.yaml_conf_setter.load(self.opensearch_yml)
         self.assertNotIn("cluster.initial_cluster_manager_nodes", opensearch_conf)
 
-        # test unicast_hosts content
```

Comment:
why was this test deleted?

Reply:
Understood, i think we can keep it and just remove the hostnames from the `expected` collections, and exclusively keep the IP